### ISSUE TITLE
OSX/macos dnnl v1.1 broken symbolic link causes file not found

### ISSUE THREAD

**Issue by dcslin** [intel/mkl-dnn#670]
```
lrwxr-xr-x   1 sc  staff        17 Mar 15 12:16 libdnnl.1.dylib -> libdnnl.1.1.dylib
lrwxr-xr-x   1 sc  staff        15 Mar 15 12:16 libdnnl.dylib -> libdnnl.1.dylib
-rwxr-xr-x   1 sc  staff  22132964 Mar 15 12:16 libdnnl.1.1.dylib
lrwxr-xr-x   1 sc  staff        15 Mar 15 12:16 libmkldnn.dylib.1 -> libdnnl.dylib.1
lrwxr-xr-x   1 sc  staff        17 Mar 15 12:16 libmkldnn.dylib.1.1 -> libdnnl.dylib.1.1
lrwxr-xr-x   1 sc  staff        13 Mar 15 12:16 libmkldnn.dylib -> libdnnl.dylib
```

`libdnnl.dylib.1` and `libdnnl.dylib.1.1` are not found
thus `libmkldnn.dylib.1` and `libmkldnn.dylib.1.1` are not found.

much appreciated for the advice

**Comment by rsdubtso** [comment]
Hi @dcslin. Thanks for the report. Sorry you ran into this issue. It has been fixed in 1.2. So, the only advice is to upgrade: we have no plans for another 1.1 patch release.

**Comment by dcslin** [comment]
Hi @rsdubtso noted and thanks.
